### DISCUSSION TITLE
Best themes exchange

### DISCUSSION THREAD

**Discussion by ildar** [davatorium/rofi#2058] [Theming]
Rofi doesn't have bundled themes so it's quite hard to find one a user wants. I search a few places and failed. Anyone plz point me out?
I could expect to find a theme here or a Category/Tag at https://www.pling.com . Maybe it's worth making that for mere users who aren't professional web designers?
E.g. I'm looking a `drun` theme with big/huge icons. Thanks.

**Comment by DaveDavenport** [comment]
???? rofi ships `rofi-theme-selector` and several themes.  See the manpage and website.

The website even shows screenshot for all these images:

https://davatorium.github.io/rofi/themes/themes/

**Comment by ildar** [comment]
Dave, thanks so much. I indeed mislooked that page.
But really, people always want their own little peculiarities. I'd gladly
looked through what people do besides those standard rasi-s.
But thanks.
P.S. The one that fits me best is `iggy` but with a semi-transparent bg.
That's just for example.

**Reply by DaveDavenport** [reply]
i don't have the time, energy or intention to make a 'theme website'.
I put my hobby project online, because I thought others might like it, I regret this from time to time.

> Maybe it's worth making that for mere users who aren't professional web designers?

I am **not** a web designer, let alone a professional one..   I actually hate having to do anything 'web'  with a passion. 

~~I have no idea where you are coming from with a statement like this...~~

**Reply by DaveDavenport** [reply]
Searching on github/google for a theme should be easy enough..

https://github.com/search?q=rofi+theme&type=repositories
for theme repositories others made

or to search for any rasi file
https://github.com/search?q=path%3A**%2F*.rasi&type=code

**Comment by ildar** [comment]
sorry, I didn't at all intended to offence anyone including you.
> Maybe it's worth making that for mere users who aren't professional web
designers?
That was based on a false assumption that no bundled theme is available, I
am sorry again I overlooked it. Regarding "professional web designers" it
was just a phrase expressing that making a new theme is quite hard. I tried
and that was ugly. Hence my desire to have people exchange their files.
Searching is great but not as a specialized site like the one in the OP
above.
Sorry again if my words hurt you. Didn't mean to do that. I just thought
that the "Discussions" section is ok for rough and unpolished thoughts.

**Reply by DaveDavenport** [reply]
Not hurt and no offense taken.

I cut out part of my reply before posting, after re-reading I can see how you interpreted that in the current form. Apologies.
